server/routes/contacts/index.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const router = express.Router();
+const Contact = require('../../../database/models/Contact');
+const User = require('../../../database/models/User');
+
+router.get('/contacts', (req, res) => {
+  Contact.fetchAll()
+    .then(contacts => {
+      return res.json(contacts);
+    })
+    .catch(() => {
+      res.status(500).send('Error retrieving contacts');
+    });
+});
+
+router.get('/contacts/search', (req, res) => {});
```

Comment:
You shouldn't include routes that aren't finished. If any requests were made to these routes, the request would hang the server.